nomad/structs/structs.go

```diff
@@ -3264,7 +3264,7 @@ func (n *NodeResources) Copy() *NodeResources {
 
 	// COMPAT remove in 1.9+
 	// apply compatibility fixups covering node topology
```

Comment:
@shoenig: is this instead safe to simply remove in 1.9 as suggested by this comment? It looks like we introduced this in 1.6.x?